src/components/AnimateSharedLayout/index.tsx

```diff
@@ -48,7 +48,7 @@ export class AnimateSharedLayout extends React.Component<SharedLayoutProps> {
      */
     syncContext: SharedLayoutSyncMethods = {
         ...createBatcher(),
-        syncUpdate: force => this.scheduleUpdate(force),
+        syncUpdate: () => this.scheduleUpdate(),
```

Comment:
Was force doing something before or is this just cleanup of an unused prop?

Reply:
Cleanup